playbooks/openshift-node/scaleup.yml

```diff
@@ -8,8 +8,25 @@
       msg: >
         Detected no new_workers in inventory. Please add hosts to the
         new_workers host group to add nodes.
-    when:
-    - groups.new_workers | default([]) | length == 0
+    when: groups.new_workers | default([]) | length == 0
+  - name: Get release image
+    oc_obj:
+      kubeconfig: "{{ kubeconfig_path }}"
+      state: present
+      kind: clusterversion.config.openshift.io
+      name: version
+    register: version
```

Comment:
`version` is a generic name for a variable.  I'd prefer something more greppable for finding usage in other parts of the file.

Reply:
Good idea, renamed to `clusterversion`